docs/deploy/local.md

```diff
@@ -75,11 +80,13 @@ juju add-model finos-legend
 ## Deploy the Legend Bundle
 When you deploy an application with Juju, the installation code in the charmed operator will run and set up all the resources and environmental variables needed for the application to run properly. In the case of this tutorial, we are deploying a *bundle*, which describes applications to be deployed and relationships between them.
 
-Deploy the finos-legend-bundle in the finos-legend model using the command line :
+Deploy the finos-legend-bundle in the finos-legend model using the command line:
 ``` bash
 juju deploy finos-legend-bundle --trust --channel=edge
 ```
 
+If the command fails on first attempt, try again.
+
```

Comment:
this part is not necessary, not very useful. If it fails, there could be an issue which would have to be addressed first. That is why a troubleshooting guide below would be useful.